doc/source/contributing_code.rst

```diff
@@ -450,30 +452,30 @@ Writing new code
 Code formatting in Python
 -------------------------
 
-MDAnalysis is a project with a long history and many contributors; it hasn't used a consistent coding style. Since version 0.11.0, we are trying to update all the code to conform with `PEP8`_. Our strategy is to update the style every time we touch an old function and thus switch to `PEP8`_ continuously.
+Since version 0.11.0, we are trying to update all the code to conform with `PEP8`_.
 
-**Important requirements (from PEP8)**:
-    - keep line length to **79 characters or less**; break long lines sensibly although for readability we may allow longer lines
-    - indent with **spaces** and use **4 spaces per level**
+**Important requirements (from PEP8):**
     - naming:
 
         - classes: `CapitalClasses` (i.e. capitalized nouns without spaces)
         - methods and functions: `underscore_methods` (lower case, with underscores for spaces)
 
 We recommend that you use a Python Integrated Development Environment (IDE) (`PyCharm`_ and others) or external tools like `flake8`_ for code linting. For integration of external tools with emacs and vim, check out `elpy`_ (emacs) and `python-mode`_ (vim).
 
-To apply the code formatting in an automated way, you can also use code formatters. External tools include `autopep8`_ and `yapf`_. Most IDEs either have their own code formatter or will work with one of the above through plugins.
-See :ref:`format-darker` for notes on maintaining code style compliance with existing tools.
+MDAnalysis uses `black`_` for automated code formatting, which is bound by the `black stability policy`_. The configuration for `black`_ is provided in the `pyproject.toml` files. To format your code with `black`_, you can run the following commands:
```

Comment:
Do we need a flag to make sure you use the right version of black?

Reply:
The right version is defined in the `pyproject.toml` from https://github.com/MDAnalysis/mdanalysis/pull/4725, so I don't think we need any flag here.

The correct version (i.e. matching the configuration) is pinned in the installation instructions above.

Reply:
Ok sounds good to me!